src/components/PeerSelector/PeerSelectorTs.ts

```diff
@@ -119,29 +104,38 @@ export class PeerSelectorTs extends Vue {
   public imageResources = dashboardImages
 
   /**
-  * Type the ValidationObserver refs 
-  * @type {{
-  *     observer: InstanceType<typeof ValidationObserver>
-  *   }}
-  */
+   * Type the ValidationObserver refs
+   * @type {{
+   *     observer: InstanceType<typeof ValidationObserver>
+   *   }}
+   */
   public $refs!: {
     observer: InstanceType<typeof ValidationObserver>
   }
 
   /// region computed properties getter/setter
-  get peersList(): string[] {
-    return this.knownPeers
+  get peersList(): NodeModel[] {
+    const nodeModels = this.knowNodes.filter(p => {
+      if (!this.formItems.filter) {
+        return true
+      } else {
+        return p.url.includes(this.formItems.filter)
+          || (p.friendlyName && p.friendlyName.includes(this.formItems.filter))
+      }
+    })
+    return _.sortBy(nodeModels, (a) => a.isDefault !== true, (a) => a.url)
```

Comment:
if you really want to use lodash, you would have to add it as a dependency and import it in the file

Reply:
I added the dependency before, just missed the import in this file. It fixed